news/r2s_change_loop_variables_name.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:** 
+* Use v (ves) and e(energy) to replace i, j
```

Comment:
Perhaps add a little more context here, e.g.
* Change loop variables to be `v` for volume elements and `e` for energy groups (instead of i & j)
* Use `bias_mode` instead of `mode` to allow for additional mode types in future